### PR TITLE
Add facets for CSG

### PR DESCRIPTION
**Pull request by tommyp**
Grant type, Land use, Tiers or standalone items and funding per unit per year. Also adding the inputs to the form and the indexable formatter. [Ticket](https://trello.com/c/gZoWid5B/523-finder-countryside-stewardship-grants).

### CODE REVIEW

File: app/exporters/formatters/countryside_stewardship_grant_indexable_formatter.rb [hunk 1]
```diff
@@ -7,7 +7,12 @@ def type
 
   private
   def extra_attributes
-    {}
+    {
+      grant_type: entity.grant_type,
+      land_use: entity.land_use,
+      tiers_or_standalone_items: entity.tiers_or_standalone_items,
+      funding_per_unit_per_year: entity.funding_per_unit_per_year,
```
Comment (evilstreak) [inline app/exporters/formatters/countryside_stewardship_grant_indexable_formatter.rb]: Could we call this something a bit less of a mouthful, like `funding_amount`? The per unit per year thing feels like a display concern, so something to just put into the label strings.

Comment (tommyp) [inline app/exporters/formatters/countryside_stewardship_grant_indexable_formatter.rb]: I was thinking that myself but I thought I'd put in the full label for the sake of discussing it. I'll change it now.

File: finders/schemas/countryside-stewardship-grants.json [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+    },
+    {
+      "key": "tiers_or_standalone_items",
+      "name": "Tiers or standalone items",
+      "type": "multi-select",
+      "preposition": "for",
+      "allowed_values": [
+        {"label": "Higher Tier", "value": "higher-tier"},
+        {"label": "Mid Tier", "value": "mid-tier"},
+        {"label": "Standalone capital items", "value": "standalone-capital-items"}
```
Comment (evilstreak) [inline finders/schemas/countryside-stewardship-grants.json]: Have we found out whether it's only possible to have certain combinations of these allowed values and the allowed values for grant type? I think that's something Holly was asking about.

**Comment by evilstreak** [comment]
Other than the funding amount key, this is all fine. I'd like to speak to Holly about the tiers/standalone items facet before we merge this though.

**Comment by jackscotti** [comment]
:+1:

**Pull request merged**